mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -617,13 +668,22 @@ class BackgroundTasksManager : BroadcastReceiver() {
                 val info = alarmManager.nextAlarmClock
                 val sender = info?.showIntent?.creatorPackage
                 Log.d(TAG, "Alarm sent by $sender")
+                val formatter = SimpleDateFormat("HH:mm yyyy-MM-dd", Locale.US)
+                val timeStamp = info?.triggerTime?.let { formatter.format(it) }
```

Comment:
Create the formatter inside the lambda to not needlessly create it if time or info are null?

Reply:
Done